homeassistant/components/webostv/manifest.json

```diff
@@ -1,9 +1,17 @@
 {
   "domain": "webostv",
   "name": "LG webOS Smart TV",
+  "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/webostv",
-  "requirements": ["aiopylgtv==0.4.0"],
+  "requirements": ["getmac==0.8.2", "aiopylgtv==0.4.0"],
```

Comment:
https://developers.home-assistant.io/docs/entity_registry_index/#example-acceptable-sources-for-a-unique-id   We generally don't allow getmac since it won't work unless the device is on the same vlan

Reply:
I did not find a good method to find a unique ID during the discovery.
I will remove it from the code.